prover/setup.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+cd ..
+export ZKSYNC_HOME=$(pwd)
+cd prover
+
+sed -i '' 's/^ETH_SENDER_SENDER_PROOF_SENDING_MODE.*$/ETH_SENDER_SENDER_PROOF_SENDING_MODE=OnlyRealProofs/' ../etc/env/dev.env
```

Comment:
dev.env file can be overwritten by the scripts.

the 'zk config compile dev' is taking data from dev.toml (and other places) and writes to dev.env

so I'd suggest maybe updating dev.toml, and then calling `zk config compile dev` ?

Reply:
Makes sense, i changed it here [9ea0427](https://github.com/matter-labs/zksync-era/pull/258/commits/9ea0427803f4f18e317e61ede60b008d68c65c1d)